src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,17 @@
-export const TodoList = () => {};
+import React from "react";
+import { TodoInfo } from "../TodoInfo";
+import { Todo } from "../../types/Todo";
+
+type Props = {
+  todos: Todo[]
+};
+
+export const TodoList: React.FC<Props> = ({ todos }) => {
+  return (
+    <section className="TodoList">
```

Comment:
use semantic tags, i.e `ul` for a list
```suggestion
    <ul className="TodoList">
```